app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -61,15 +62,25 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
 
         settingsPresenter = SettingsPresenter(this)
         val skillFragment = SkillListingFragment()
-        //skills = skillFragment.skills
-        supportFragmentManager.beginTransaction()
-                .add(R.id.fragment_container, skillFragment, TAG_SKILLS_FRAGMENT)
-                .addToBackStack(TAG_SKILLS_FRAGMENT)
-                .commit()
+        val privacyFragment = PrivacyFragment()
+        val bundle = intent.extras
+        val intentValueSignUpToPrivacy = bundle?.getBoolean(Constant.SIGN_UP_TO_PRIVACY)
```

Comment:
```suggestion
        val toPrivacy = bundle?.getBoolean(Constant.SIGN_UP_TO_PRIVACY)
```

Reply:
Still not changed

Reply:
Sir, can you please suggest what name shall I give here?

Reply:
I already have, look above